lib/index.js

```diff
@@ -12,7 +12,12 @@ var ParserInline = require('./parser_inline');
 var LinkifyIt    = require('linkify-it');
 var mdurl        = require('mdurl');
 var punycode     = require('punycode');
+var trimRight    = require('trim-right');
 
+if (!String.prototype.trimRight) {
```

Comment:
> I'd like to avoid microdependencies and prototype changes

Ok - removed the trim part entirely.

> PS. we can fix tables bug ourself after weekend.

The you mean the truncation of cells from tbody rows that are longer than the thead row? I don't mind waiting till Monday if you're busy... switching tidy-markdown over to a new compiler is just a side-project rather than something with a schedule.